panic: runtime error: invalid memory address or nil pointer dereference

## Description

<!--
Briefly describe the problem you are having in a few paragraphs.
-->
`/usr/local/bin/trivy rootfs  /` -- command dies with SIGSEGV.

## What did you expect to happen?
It should "just work".

## What happened instead?
It "just bombed", after 3.5 minutes.

## Output of run with `-debug`:

```
...
2022-04-03T18:01:19.863-0700	DEBUG	No such POM in the central repositories	{"file": "url-test.jar"}
2022-04-03T18:01:19.892-0700	DEBUG	Parsing Java artifacts...	{"file": "home/user/gradle/wrapper/gradle-wrapper.jar"}
2022-04-03T18:01:19.928-0700	DEBUG	No such POM in the central repositories	{"file": "lf.jar"}
2022-04-03T18:01:20.404-0700	DEBUG	No such POM in the central repositories	{"file": "gradle-wrapper.jar"}
2022-04-03T18:01:20.936-0700	DEBUG	Analysis error: unable to decode JSON (home/user/catapult/telemetry/telemetry/internal/backends/chrome/chromeos_login_ext/manifest.json): invalid character '/' looking for beginning of object key string
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x15cba0a]

goroutine 60930 [running]:
github.com/aquasecurity/fanal/analyzer/language/python/packaging.packagingAnalyzer.Analyze({}, {0x1c8be20?, 0x7020636f44534a00?}, {{0x7ffcad3c34f5, 0x1}, {0xc015b53d41, 0x83}, {0x2192590, 0xc015b51930}, {0x218c580, ...}, ...})
	/home/runner/go/pkg/mod/github.com/aquasecurity/fanal@v0.0.0-20220324154234-b2df5b98f8cd/analyzer/language/python/packaging/packaging.go:53 +0x18a
github.com/aquasecurity/fanal/analyzer.AnalyzerGroup.AnalyzeFile.func1({0x218dad0, 0x3102790}, {0x218f2e0?, 0xc000b7d4b8})
	/home/runner/go/pkg/mod/github.com/aquasecurity/fanal@v0.0.0-20220324154234-b2df5b98f8cd/analyzer/analyzer.go:253 +0x253
created by github.com/aquasecurity/fanal/analyzer.AnalyzerGroup.AnalyzeFile
	/home/runner/go/pkg/mod/github.com/aquasecurity/fanal@v0.0.0-20220324154234-b2df5b98f8cd/analyzer/analyzer.go:248 +0x3d8

```

## Output of `trivy -v`:

```
Version: 0.25.0
Vulnerability DB:
  Version: 2
  UpdatedAt: 2022-04-04 00:09:56.817083217 +0000 UTC
  NextUpdate: 2022-04-04 06:09:56.817082817 +0000 UTC
  DownloadedAt: 2022-04-04 00:56:46.423024596 +0000 UTC

```

## Additional details (base image name, container registry info...):
Ubuntu 18.04.6 LTS

--- a/go.mod
+++ b/go.mod
@@ -7,7 +7,7 @@ require (
 	github.com/Masterminds/sprig/v3 v3.2.2
 	github.com/NYTimes/gziphandler v0.0.0-20170623195520-56545f4a5d46
 	github.com/aquasecurity/bolt-fixtures v0.0.0-20200903104109-d34e7f983986
-	github.com/aquasecurity/fanal v0.0.0-20220413153130-ae71d24c572b
+	github.com/aquasecurity/fanal v0.0.0-20220414083417-61bfa9f92483
 	github.com/aquasecurity/go-dep-parser v0.0.0-20220412145205-d0501f906d90
 	github.com/aquasecurity/go-gem-version v0.0.0-20201115065557-8eed6fe000ce
 	github.com/aquasecurity/go-npm-version v0.0.0-20201110091526-0b796d180798

